corehq/ex-submodules/couchforms/models.py

```diff
@@ -346,7 +346,7 @@ def _meta_to_json(meta):
     def xml_md5(self):
         return hashlib.md5(self.get_xml()).hexdigest()
 
-    def archive(self, user_id=None):
+    def archive(self, user_id=None, skip_rebuild_cases=False):
```

Comment:
I think this argument was slightly easier to understand without the double-negative (i.e., `rebuild_cases=True` was better).

Reply:
okay.. reverted and ensured we do exact check
https://github.com/dimagi/commcare-hq/pull/24441/commits/8588bf413f453f8df765eb3572f23381aa1cc18b
https://github.com/dimagi/commcare-hq/pull/24441/commits/d2c7e06d5c57110cf00188062766d8cfdddb87fd
https://github.com/dimagi/commcare-hq/pull/24441/commits/d5a2a1ae6c95bd9e06ff0dd9e1baf2cff4c59413

Reply:
https://github.com/dimagi/commcare-hq/pull/24441/commits/469ccefd73adfe7d591b36bf7885359d4ff7dfcf